src/state.rs

```diff
@@ -161,6 +164,68 @@ impl MemberState {
     }
 }
 
+impl Debug for MemberState {
+    fn fmt(&self, formatter: &mut Formatter) -> fmt::Result {
```

Comment:
Why do you need that do pretty assert not do the trick?

Reply:
Oh, no - it just uses the `{:#?}` style I think which is much worse to read than `{:?}` IMO.  Probably good for smaller structs.  It also uses the same colour of green for background and text in one of my consoles, making the text invisible.  I actually didn't add the pretty_assert macro to the new test file.

I'd be happy to remove this though and keep a copy locally for debugging.

Reply:
Can you indicate where/how we want to use this MemberState print?

Reply:
I've only used it for temporary debugging.

After discussion IRL, I'll remove the multiple `Debug` implementations for the types in utilities.rs, handle those inside this method, and change this to a `Display` implementation.

Reply:
Sounds like multiple chairs were thrown before reaching an agreement ;)

Reply:
How would you feel about a single `Debug` implementation in utilities.rs for `Attributes` @jeanphilippeD ?  It makes a big difference to many types.  Without this:

```
Node(Attributes { age: Age(18), name: Name(-1423112477) })
Candidate(Attributes { age: Age(18), name: Name(-1423112477) })
```
and by implementing `Debug for Attributes` as `write!(formatter, "{:?}, {:?}", self.age, self.name)` we get:
```
Node(Age(18), Name(-1423112477))
Candidate(Age(18), Name(-1423112477))
```

Reply:
I think I can live with that :-)

Reply:
Excellent! :)  I'll put the chair down now!

Thanks!